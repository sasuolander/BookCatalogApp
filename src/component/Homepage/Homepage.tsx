import React from "react";

export const Homepage = ({}) => {
  return (
    <div>
      <p>home page</p>
    </div>
  );
};
