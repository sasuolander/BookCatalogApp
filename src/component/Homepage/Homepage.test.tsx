import React from 'react'
import { Homepage } from "./Homepage";
import { mount, shallow, render } from "enzyme";

it("render component homepage",()=>{
    const component = shallow(<Homepage/>)
    expect(component).toMatchSnapshot();
})