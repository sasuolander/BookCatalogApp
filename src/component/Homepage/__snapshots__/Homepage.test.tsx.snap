// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render component homepage 1`] = `
<div>
  <p>
    home page
  </p>
</div>
`;
